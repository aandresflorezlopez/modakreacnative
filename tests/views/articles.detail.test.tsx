import { screen, render } from '@testing-library/react-native';
import { waitFor } from '@testing-library/dom';

import ArticleDetail from '../../src/views/article.detail';

describe('[Article detail]', () => {
  test('no renders article detail correctly', () => {
    render(<ArticleDetail />);

    expect(screen.toJSON()).toMatchSnapshot();
  });
});
