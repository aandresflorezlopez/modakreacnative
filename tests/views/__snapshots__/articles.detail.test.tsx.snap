// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Article detail] no renders article detail correctly 1`] = `
<View
  collapsable={false}
  style={
    {
      "alignSelf": "center",
      "flex": 1,
      "justifyContent": "center",
      "opacity": 0,
    }
  }
>
  <Image
    source={
      {
        "testUri": "../../../assets/waiting.png",
      }
    }
    style={
      [
        {
          "height": 50,
          "paddingtop": "10em",
          "pagging": "20em",
          "width": 50,
        },
      ]
    }
  />
</View>
`;

exports[`[Article detail] renders article detail correctly 1`] = `
<View
  collapsable={false}
  style={
    {
      "alignSelf": "center",
      "flex": 1,
      "justifyContent": "center",
      "opacity": 0,
    }
  }
>
  <Image
    source={
      {
        "testUri": "../../../assets/waiting.png",
      }
    }
    style={
      [
        {
          "height": 50,
          "paddingtop": "10em",
          "pagging": "20em",
          "width": 50,
        },
      ]
    }
  />
</View>
`;
